### PR TITLE
chore: Added test for loading json and interblock deletion

### PR DESCRIPTION
**Pull request by ericblackmonGoogle**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)
- [x] I ran `npm run format` and `npm run lint`

### Proposed Changes

Added testing for loading blocks and large interconnected block deletion.

### CODE REVIEW

File: tests/browser/test/extensive_test.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+const {
+  testSetup,
+  testFileLocations,
+  getBlockTypeFromWorkspace,
+} = require('./test_setup');
+const {Key} = require('webdriverio');
+
+let browser;
+suite(
+  'Testing Blockly Loading Large Configurations and Deletion',
```
Comment (BeksOmega) [inline tests/browser/test/extensive_test.js:21]: Can you change your suite name to "Loading Large Configuration and Deletion"?

Generally for test suites I like to follow the convention of "This tests X" where X becomes the suite title!

**Review by BeksOmega — COMMENTED** [review]
Structure looks great! Mostly just some nits about naming, and a suggestion about accessing blocks!

**Comment by ericblackmonGoogle** [comment]
All changes have been made!

**Review by BeksOmega — APPROVED** [review]
Just one more nit then feel free to merge =)

**Pull request merged**